components/dashboard/src/teams/GitIntegrationsPage.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FunctionComponent, useMemo } from "react";
+import { Redirect } from "react-router";
+import Header from "../components/Header";
+import { SpinnerLoader } from "../components/Loader";
+import { useOrgBillingMode } from "../data/billing-mode/org-billing-mode-query";
+import { useCurrentOrganizationMember } from "../data/organizations/org-members-query";
+import { GitIntegrations } from "./git-integrations/GitIntegrations";
+import { OrgSettingsPage } from "./OrgSettingsPage";
+
+export default function GitAuth() {
+    return (
+        <OrgSettingsPageWrapper>
+            <GitIntegrations />
+        </OrgSettingsPageWrapper>
+    );
+}
+
+// TODO: Refactor this into OrgSettingsPage so each page doesn't have to do this
+export const OrgSettingsPageWrapper: FunctionComponent = ({ children }) => {
```

Comment:
I initially had this in a version of what is now the `<OrgSettingsPage/>` so that loading state and role checks across the settings pages are in one place. I didn't want to push this change across the other settings pages in this PR, so kept it here for now. I'll follow up with shifting this into the `<OrgSettingsPage/>` component in a subsequent PR.

Reply:
That makes a lot of sense. Thanks so much for mowing the lawn 🙏🏻